pom.xml

```diff
@@ -73,6 +73,19 @@
     </developer>
   </developers>
 
+  <distributionManagement>
+    <repository>
+      <id>central</id>
+      <name>AppsFlyer Artifactory-releases</name>
+      <url>https://artifactory.appsflyer.com:443/artifactory/maven</url>
+    </repository>
+    <snapshotRepository>
+      <id>snapshots</id>
+      <name>AppsFlyer Artifactory-snapshots</name>
+      <url>https://artifactory.appsflyer.com/artifactory/maven-snapshots/</url>
```

Comment:
@GrigorievNick Just curious, do we need any artifact from snapshots?

Reply:
@cheeseng I am sorry I just deploy this to my own artifactory.
and accidentally commit custom pom details.